cmd/root.go

```diff
@@ -115,7 +115,7 @@ func loadConfiguration() (*config.Config, error) {
 	k8sCfg := client.NewConfig(k8sFlags)
 	k9sCfg := config.NewConfig(k8sCfg)
 	if err := k9sCfg.Load(config.AppConfigFile); err != nil {
-		return nil, err
+		log.Error().Err(err).Msgf("load configuration failed")
```

Comment:
I think we should panic if the base configuration is wrong. So that users can address it?

For the missing config part it should have been generated if the file does not exist. I've missed that ;(